packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx

```diff
@@ -16,6 +16,9 @@ import {
   StyledSubNavItemIcon
 } from '../../styled';
 
+/**
+ * @extends ButtonHTMLAttributes<HTMLButtonElement>
+ */
```

Comment:
I think this needs to be moved to the `CollapsibleSubNavItem` export below and changed to `HTMLAttributes<HTMLDivElement>` since that forwards a ref to a div element.